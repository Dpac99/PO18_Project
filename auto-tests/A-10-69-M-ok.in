100010
1
e1.obj
5
1
D1
P5
eee
0
6
6
D1
0
2
0
