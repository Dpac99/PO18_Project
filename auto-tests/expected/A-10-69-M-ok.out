Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
1 - Entregar projecto
2 - Preencher inquérito
3 - Ver resultados de inquérito
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: D1 - P0 (por abrir)
D1 - P1 - 1 respostas - 1 horas
D1 - P2 (aberto)
D1 - P4 (aberto)
D1 - P5 (por abrir)
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: 
